prow/git/git.go

```diff
@@ -245,6 +245,11 @@ func (r *Repo) gitCommand(arg ...string) *exec.Cmd {
 	return cmd
 }
 
+// Run allows to execute arbitrary git commands
```

Comment:
Why should we expose this instead of explicit client methods? I worry that it's too easy to break some "infra" with a black-box `Run()` and impossible to review usage.